pkg/distsql/distsql.go

```diff
@@ -84,6 +87,22 @@ func Select(ctx context.Context, dctx *distsqlctx.DistSQLContext, kvReq *kv.Requ
 		TryCopLiteWorker:           &dctx.TryCopLiteWorker,
 	}
 
+	// Force the CopLiteWorker to be used or not used for testing purposes
+	failpoint.Inject("TryCopLiteWorker", func(val failpoint.Value) {
+		var n int
+		var ok bool
+		if n, ok = val.(int); !ok {
+			panic(fmt.Sprintf("TryCopLiteWorker: expected int, got %T (%v)", val, val))
+		}
+
+		v := atomic.NewUint32(uint32(n))
```

Comment:
Seems you can directly use `option.TryCopLiteWorker.Store(uint32(n))`, no need to create `v`